sspl_test/functional_tests/framework/utils/config_reader.py

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import os
+import sys
+import pickle
+import configparser
+from framework.base.sspl_constants import (
+    component,
+    file_store_config_path,
+    SSPL_STORE_TYPE,
+    StoreTypes,
+    CONSUL_HOST,
+    CONSUL_PORT,
+)
+from framework.utils.service_logging import logger
+
+
+# Onward LDR_R2, consul will be abstracted out and won't exist as hard dependency for SSPL
+if SSPL_STORE_TYPE == "consul":
+    import consul
+
+
+class ConfigReader(object):

```

Comment:
again, check why we cant use SSPL config reader class code here directly ? Lot of code duplication here